codecov.yml

```diff
@@ -10,9 +10,5 @@ coverage:
   status:
     project:
       default:
-        threshold: 0.25%
+        target: 100%
```

Comment:
Similarly, it's a little silly to set a target of 100% coverage. But again, this library is _complex_, and (thus far) can be written without any runtime assertions that would prove difficult to test.

Given how fundamental this repo will be for consumers, let's start out by holding ourselves to a high standard.